src/components/Header/HeaderUserMenu/HeaderUserMenu.js

```diff
@@ -62,6 +69,27 @@ const HeaderUserMenu = ({ user }) => {
             ''
           )}
         </div>
+        {user.sms.description ? (
```

Comment:
Perhaps here instead of ask for a property this way, how about doing this?
```suggestion
        {Object.keys(user.sms).length ? (
```